helpers/calendar/analytics.py

```diff
@@ -162,7 +37,7 @@ def get_monthly_rooms_events(self, query, month, year):
             else:
                 for event in calendar_events:
                     if event.get('attendees'):
-                        event_details = RoomAnalytics.get_event_details(
+                        event_details = CommonAnalytics.get_event_details(
```

Comment:
Any reason for renaming the class to Common Analytics?

Reply:
@shemogumbe I moved the reusable functions to another file ie analytics_helper.py under a class called Common Analytics and 'get_event_details' is one of those functions that I moved so as to reduce the number lines in the analytics.py file